### DISCUSSION TITLE
click hyperlink to open node

### DISCUSSION THREAD

**Discussion by xmatic** [freeplane/freeplane#1693] [General]
Let's say I have a node that contains the following text.

> A
> B

**What I want:** when I click on A, it opens a node related to A, and when I click on B, it opens a node related to B.

What I know: 
1. how to add hyperlinks to different pieces of text in a node
2. how to copy a nodeID (apparently, when linking to a node using a hyperlink, one can use nodeID in lieu of a URL)

Your thoughts and help are appreciated. Thank you in advance.

**Comment by euu2021** [comment]
Use a `#` before the node ID, like this:
![image](https://github.com/freeplane/freeplane/assets/77707706/aae38ca9-ff29-4f34-b3c7-dd678f126d55)

**Comment by xmatic** [comment]
> Use a `#` before the node ID, like this: ![image](https://private-user-images.githubusercontent.com/77707706/304737737-aae38ca9-ff29-4f34-b3c7-dd678f126d55.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDc5MzYwNDIsIm5iZiI6MTcwNzkzNTc0MiwicGF0aCI6Ii83NzcwNzcwNi8zMDQ3Mzc3MzctYWFlMzhjYTktZmYyOS00ZjM0LWIzYzctZGQ2NzhmMTI2ZDU1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMTQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjE0VDE4MzU0MlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTYzZjA4OTc5ODczYjQyZmE4ZGI5YzllMjZiOWU1OTE4MmZkZTg1YjljMzFjN2ZmZGEyNWI2N2JmNDdlMWVmNDYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.y-ta_sTxk-b58p50Ivg5aNMAVwVVh1XZP_nvATn288E)

Thank you. That helped!

While you're reading this: Any way to only have the relevant child node open? Currently, all the child nodes open at once.

**Reply by euu2021** [reply]
> While you're reading this: Any way to only have the relevant child node open? Currently, all the child nodes open at once.

I don't think so. This makes me remember this request: https://github.com/freeplane/freeplane/issues/1224

**Comment by EdoFro** [comment]
Hi @xmatic,
You can also look into the documentation map: 

Menu `Help->Documentation (F1)`

and there look at topic **"5. Hyperlinking"**. It has some nice information. 

Take a look at:
- the Alias section
   - understand what are aliases and global nodes
- the path section
   - this is a lost art very few people talk about, but it seems it is what you are looking for

![image](https://github.com/freeplane/freeplane/assets/35700575/14f13bc3-83ce-4651-90e3-65eeb29679db)

hope it helps

**Comment by xmatic** [comment]
> Hi @xmatic, You can also look into the documentation map:
> 
> Menu `Help->Documentation (F1)`
> 
> and there look at topic **"5. Hyperlinking"**. It has some nice information.
> 
> Take a look at:
> 
> * the Alias section
>   
>   * understand what are aliases and global nodes
> * the path section
>   
>   * this is a lost art very few people talk about, but it seems it is what you are looking for

Thank you! Will look into this!

**Comment by EdoFro** [comment]
I made myself a mindmap to test the "linking nodes by path" concept.

I added some notes and scripts that may be usefull to others:

![image](https://github.com/freeplane/freeplane/assets/35700575/b2cb3b9c-3d07-4472-b26c-5694b024d3c9)

each link path text can be seen in the node's details.
In the node's note is also an explanation of it.

here is the example mindmap:
[linking nodes.zip](https://github.com/freeplane/freeplane/files/14289526/linking.nodes.zip)

hope it helps,

edo

**Reply by xmatic** [reply]
Thank you for being so generous with your help.
